tests/integration/pilot/vm/main_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vm
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework/label"
+
+	"istio.io/istio/pkg/test/framework/components/namespace"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/components/pilot"
+	"istio.io/istio/pkg/test/framework/resource"
+)
+
+var (
+	i  istio.Instance
+	p  pilot.Instance
+	ns namespace.Instance
+)
+
+// This post-submit suite tests VM mesh expansion. Rather than deal with the infra to get a real VM, we will use a pod
+// with no Service, no DNS, no service account, etc to simulate a VM.
+func TestMain(m *testing.M) {
+	framework.
+		NewSuite(m).
+		Label(label.Postsubmit).
```

Comment:
we should run a single VM in presubmit and then the rest in postsubmit probably. Otherwise we will break stuff all the time without noticing

Reply:
You are right. I will fix this.